client/web/compose/src/components/Common/Grid.vue

```diff
@@ -19,25 +19,29 @@
       :use-css-transforms="false"
       @layout-updated="handleLayoutUpdate"
     >
-      <grid-item
+      <template
         v-for="(item, index) in gridCollection"
-        :key="item.i"
-        ref="items"
-        class="grid-item"
-        :class="{
-          'h-100': isStretchable,
-        }"
-        style="touch-action: none;"
-        v-bind="{ ...item }"
       >
-        <slot
-          :block="blocks[item.i]"
-          :index="index"
-          :block-index="item.i"
-          :bounding-rect="boundingRects[index]"
-          v-on="$listeners"
-        />
-      </grid-item>
+        <grid-item
+          v-if="blocks[item.i].options.tabbed !== true"
```

Comment:
We can do that but I believe this is better because there are many instances where we want to know if a block is tabbed anyway so why not use that property to make things easy to read. 

Secondly, the builder logic is already written in a way where there is no consistency to what the SOURCE OF TRUTH is. For example, Grid here is receiving page.blocks, but things like delete mutates blocks. When we filter page.blocks, anything we do on blocks is not needed. This helps skip over going through the code there and risking breaking anything.